docs/bin/plugin_formatter.py

```diff
@@ -338,7 +338,7 @@ def print_modules(module, category_file, deprecated, options, env, template, out
     if module in deprecated:
         modstring = modstring + DEPRECATED
 
-    category_file.write("  %s - %s <%s_module>\n" % (to_bytes(modstring), to_bytes(rst_ify(module_map[module][1])), to_bytes(modname)))
+    category_file.write("  %s - %s <%s_module>\n" % (to_text(modstring), to_text(rst_ify(module_map[module][1])), to_text(modname)))
```

Comment:
It's opened as `open(category_file_path, "w")`

In fact, we do a lot of file writing throughout, all opened as `"w"`, but without any specific indication of `to_bytes`